x-pack/legacy/plugins/ingest/common/utils/is_version_greater.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function isVersionGreater(v1: string, v2: string): 1 | 0 | -1 {
```

Comment:
seems to be unused

Reply:
Should be using this not the one in the lib as we will need it UI side, thats my bad